controller/api/urls.py

```diff
@@ -45,7 +45,7 @@
     url(r"^apps/(?P<id>{})/containers/?".format(settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'get': 'list'})),
     # application domains
-    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\._\w]+)/?".format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\._\w*]+)/?".format(settings.APP_URL_REGEX),
```

Comment:
That may be.  I'd defer to one of our resident python experts on that.  @bacongobbler